core/tests/webdriverio_utils/workflow.js

```diff
@@ -209,6 +209,53 @@ var createAndPublishTwoCardExploration = async function(
   await publishExploration();
 };
 
+var createAndPublishExplorationWithAdditionalCheckpoints = async function(
+    title, category, objective, language, welcomeModalIsShown,
+    correctnessFeedbackIsEnabled) {
+  await createExploration(welcomeModalIsShown);
+  var explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+  var explorationEditorMainTab = explorationEditorPage.getMainTab();
+  await explorationEditorMainTab.setContent(await forms.toRichText('card 1'));
```

Comment:
@ShivamJhaa do we need to pass `expectFadeIn=true` here (and elsewhere in this file)?

Reply:
No. We pass `expectFadeIn=true` when we first set the statename and after that we set content in the next step.